### DISCUSSION TITLE
Is there any API or way which would return parent workflow  ID based on child workflow ID?

### DISCUSSION THREAD

**Discussion by diti-nz** [Netflix/conductor#3015] [Q&A] [answered]
I explored the APIs from Netflix Conductor documentation page, however, I wasn't able to pick it apart and find the solution. 

Problem Statement : 
I have a nested workflow and  have a child workflow id and now I want to fetch the parent workflow id using  child workflow id .  

Does anyone have any info on this that can help me?

**Comment by v1r3n** [comment, marked as answer]
See this:
https://github.com/Netflix/conductor/blob/main/common/src/main/java/com/netflix/conductor/common/run/Workflow.java#L70

You can fetch the child workflow by id and look at the parentWorkflowId field.

**Reply by diti-nz** [reply]
Thanks @v1r3n
